ping.go

```diff
@@ -143,6 +143,7 @@ type Pinger struct {
 	avgRtt    time.Duration
 	stdDevRtt time.Duration
 	stddevm2  time.Duration
+	statsmu   sync.RWMutex
```

Comment:
Minor nit about Go naming style, this should be CamelCase. How about`statusMtx`?

Reply:
`statsMu`  ?

Most examples I see use `Mu`, not `Mtx`

Reply:
`Mu` is fine too.